### PR TITLE
Level one map editing

### PR DESCRIPTION
**Pull request by Guysus**
Cave, Desert, Beach and forest area

### CODE REVIEW

**Review by Saretzky — CHANGES_REQUESTED** [review]
What is all the added files in the _ExternalActors_ folder? is it necessary to merge?

fix the flat rocks
image

add normal map to the sand beach texture, it looks too flat. and instead of using a plane with the texture on it, paint the landscape with the material.

Blend the transition between biomes (make it look less drastic, the beach is very harsh but the transition between the other 2 biomes need to be blended more)
image

add more details in this biome (rocks, grass, plants etc)
image

**Review by Saretzky — CHANGES_REQUESTED** [review]
also the transition between the water and land need to be worked on 
image

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]
Truthfully dont know what to say, we can discuss in class

**Review by Saretzky — CHANGES_REQUESTED** [review]
review if whats done so far: theres still a lot of the above changes that needs to be done

**Review by Saretzky — CHANGES_REQUESTED** [review]
MSPresets folder showed up again

the water to beach needs some work, the line between them is too harsh
image
 
move everything up or add a proper sky box so we cant see the black void when looking at horizon
image

dont use a plain with a material on it for this. it looks stretched and flat
image

same with this area
image

there's no grass/something to fill out this area
image

what happened to the lighting in the cave?
image

**Review by AMacLeod99 — APPROVED** [review]

**Review by Saretzky — CHANGES_REQUESTED** [review]
can we put these assets in a folder
image

keep the level manager outside of a folder, just assets
image

File: Config/DefaultEngine.ini
Comment (Saretzky) [inline Config/DefaultEngine.ini:1]: change this back to hub

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by Jeff-Moreau — APPROVED** [review]
Looks Good

**Review by Saretzky — APPROVED** [review]
fixed some small stuff myself, good to go

**Pull request merged**